src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const rows = [...table.rows];
+
+rows.forEach((row) => {
+  const copy = row.cells[1].cloneNode(true);
+
+  row.cells[4].before(copy);
+});
```

Comment:
Ensure that each row has at least five cells before attempting to access `row.cells[4]`. If any row has fewer than five cells, this code will throw an error. Consider adding a check to ensure the row has enough cells before performing the operation.